zipline/algorithm.py

```diff
@@ -1031,12 +1200,34 @@ def _can_order_asset(self, asset):
 
     @api_method
     @disallowed_in_before_trading_start(OrderInBeforeTradingStart())
-    def order(self, asset, amount,
+    def order(self,
+              asset,
+              amount,
               limit_price=None,
               stop_price=None,
               style=None):
-        """
-        Place an order using the specified parameters.
+        """Place an order.
+
+        Parameters
+        ----------
+        asset : Asset
+            The asset that this order is for.
+        amount : int
+            The amount of shares to order. If this is negative, this is the
+            number of shares to sell or short.
```

Comment:
sorry, how do you want me to change the wording. I do mention that negative means short